eng/pipelines/coreclr/jitstressregs.yml

```diff
@@ -20,24 +20,31 @@ jobs:
   parameters:
     jobTemplate: /eng/pipelines/common/build-coreclr-and-libraries-job.yml
     buildConfig: checked
-    platformGroup: all
+    platforms:
+    - Linux_arm
+    - Linux_arm64
+    - Linux_x64
+    - Windows_NT_x64
+    - Windows_NT_x86
+    # Currently no Windows arm32, Windows arm64 hardware in Helix. https://github.com/dotnet/runtime/issues/1097
+    #- Windows_NT_arm
+    #- Windows_NT_arm64
     jobParameters:
       testGroup: jitstressregs
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
+    jobTemplate: /eng/pipelines/coreclr/templates/test-job.yml
     buildConfig: checked
-    platformGroup: all
-    jobParameters:
-      testGroup: jitstressregs
-      liveLibrariesBuildConfig: Release
-
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/run-test-job.yml
-    buildConfig: checked
-    platformGroup: all
+    platforms:
```

Comment:
It wasn't clear to me why jitstressregs.yml differed from the others by directly invoking build-test-job.yml and run-test-job.yml instead of using test-job.yml, so I changed it. Is this ok?

Reply:
I think that's fine.